lib/node_modules/@stdlib/math/base/special/acoth/docs/repl.txt

```diff
@@ -0,0 +1,28 @@
+
+{{alias}}( x )
+    Computes the inverse hyperbolic cotangent of a number.
+
+    If `x >= -1 && x <= 1`, the function returns `NaN`.
+
+    Parameters
+    ----------
+    x: number
+        Input value.
+
+    Returns
+    -------
+    y: number|out
+        Inverse hyperbolic cotangent (in radians).
+
+    Examples
+    --------
+    > var y = {{alias}}( 1.0 )
+    0
```

Comment:
```suggestion
    0.0
```

I believe, based on the documentation, that this should be `NaN`, not `0.0`.

Reply:
NaN is for values in between -1 and 1. for these exact values, the javascript implementation return Infinity and I set this to 0 so it is more consistent with other functions.

Reply:
For clarity, set to `0` to be consistent with what functions?